### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by ryjones**
Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- https://github.com/martmartinez91 (maintainer)
+- https://github.com/dhuseby (maintainer)
+- https://github.com/ahmetturkmen
+- https://github.com/veyselpehlivan
+- https://github.com/tapasweni-pathak
+
+# Sponsor
+- mwagner at redhat dot com - Mark Wagner
+
+# Pre-existing repository
```
Comment (tkuhrt) [inline README.md]: I normally don't include this.

Comment (ryjones) [reply README.md]: those four lines specifically?

Comment (tkuhrt) [reply README.md]: No...the "Pre-existing repository" section.

Comment (ryjones) [reply README.md]: I removed that stanza

**Comment by ryjones** [comment]
thank you, @tkuhrt

**Pull request merged**